plugins/kubernetes/app/decorators/build_decorator.rb

```diff
@@ -5,4 +5,31 @@
   def project_name
     @project_name ||= project.name.parameterize('-')
   end
+
+  def import_kubernetes_roles
+    # TODO : Build -> KubernetesRoles association
+    project.roles.destroy_all
```

Comment:
This is dangerous. `Kubernetes::ReleaseDoc` belongs_to role.  If you destroy all the roles, either the ReleaseDocs will get deleted, orphaned, or with `kubernetes_role_ids` columns pointing to a null row, or the wrong row.

Maybe some logic that checks if the project has any Kubernetes releases, and fails if it does?